src/Core/Resolvers/IQueryBuilder.cs

```diff
@@ -64,6 +64,12 @@ public interface IQueryBuilder
         /// <returns></returns>
         public string BuildStoredProcedureResultDetailsQuery(string databaseObjectName);
 
+        /// <summary>
+        /// Builds the query to determine the number of enabled triggers on a database table.
+        /// Needed only for MsSql.
+        /// </summary>
+        string BuildFetchEnabledTriggersQuery() => throw new NotImplementedException();
```

Comment:
can be renamed to `BuildMsSqlFetchEnabledTriggersCountQuery()`
It will make it clear that the method is specific to MsSQL Server, and that it returns a SQL query to fetch the count of enabled triggers on a database table.

Reply:
DB specific method names aren't appropriate for the generic interface. At some point PG / MySQL may need to utilize this. Even if not, a notImplementedException or other more verbose messaging and graceful error handling can be performed.